.github/workflows/windows.yml

```diff
@@ -290,8 +291,8 @@ jobs:
 
       - name: Install Python API tests dependencies
         run: |
-          # For torchvision to OpenVINO preprocessing converter
-          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/src/openvino/preprocess/torchvision/requirements.txt
+          # Install test-related requirements
+          python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/requirements_test.txt
```

Comment:
This is needed for `pytest-xdist` so that `-n` option is available for `pytest`. I will see how much it wins it terms of execution times for Python tests with experiments.

Reply:
The experiments were quite underwhelming. It seems like the tests themselves are not ready for this *easy* parallelization:
* There are I/O operations on models/resources and if parallelization is used - they clash.
* Those tests that have passed were slower than in the current master w/o parallelization.

Reply:
ok, agree